migrations/drop_everything._sql

```diff
@@ -0,0 +1,11 @@
+DROP TYPE note_type CASCADE;
+
```

Comment:
Its so you never accidentialy run it. For example i could imagine someone (e.g. Evan) run all migrations that end in .sql. This would accidentally get run. If you'd like i could change it to .txt. But i thought or .sql.txt.

Reply:
another option might be to put it in a separate folder? or a subfolder?